src/openzaak/components/catalogi/admin/zaaktypen.py

```diff
@@ -307,6 +323,15 @@ def get_related_objects(self, obj):
 
         return resource_list, id_list
 
+    def response_post_save_change(self, request, obj):
+
+        if "_publish" in request.POST:
```

Comment:
I think it should be a link, not a submit button now. And it should be excluded from submit process. Because when you click on this button you are redirected on the publish page with this message: 
![image](https://github.com/open-zaak/open-zaak/assets/47745906/7e011a4f-1ded-4765-b6eb-3c692014fb79)